freshmaker/image_verifier.py

```diff
@@ -183,21 +179,22 @@ def verify_repository(self, repo_name):
             "images": {}
         }
 
-        images = self.lb.find_images_with_included_rpms(
-            [], [], {repo["repository"]: repo}, include_rpm_manifest=False)
+        images = self.pgql.find_images_by_repository(repo)
```

Comment:
@qixiang Images returned by this API have different format than images returned by `find_images_with_included_rpms` in LighBlue. In `find_images_with_included_rpms`, the images are post-processed a few times beginning here https://github.com/redhat-exd-rebuilds/freshmaker/blob/pyxis/freshmaker/lightblue.py#L1011 .  Should I implement such postprocessing to pyxis as well, since the output in pyxis isn't really similar without this postprocessing that happens in LightBlue?

Reply:
Yes, we'd keep the same data structure in pyxis_gql module as what pyxis returns, in lightblue.py the images returned by LightBlue is also a list of dict, but converted to `ContainerImage` and `ContainerRepository`, so we also need to convert the result returned by pyxis_gql APIs to such objects in `lightblue.py` instead of `pyxis_gql.py`